x/accounts/defaults/lockup/continuous_locking_account.go

```diff
@@ -66,12 +66,6 @@ func (cva *ContinuousLockingAccount) Delegate(ctx context.Context, msg *lockupty
 	return cva.BaseLockup.Delegate(ctx, msg, cva.GetLockedCoinsWithDenoms)
 }
 
-func (cva *ContinuousLockingAccount) Undelegate(ctx context.Context, msg *lockuptypes.MsgUndelegate) (
-	*lockuptypes.MsgExecuteMessagesResponse, error,
-) {
-	return cva.BaseLockup.Undelegate(ctx, msg)
-}
-
```

Comment:
I might be missing context, can you explain why we don't need this here?

Reply:
Ah is it because we are now registering the handler from the baselockup directly?

Reply:
yes, since all of this are the same with other types of lockup so i move it to base lockup